src/main/java/com/teamscale/upload/CommandLine.java

```diff
@@ -204,6 +221,10 @@ public static CommandLine parseArguments(String[] args) {
 						+ " 00:00:00 UTC Thursday, 1 January 1970 or the string 'HEAD' to upload to"
 						+ " the latest revision on that branch." + "\nFormat: BRANCH:TIMESTAMP"
 						+ "\nExample: master:1597845930000" + "\nExample: develop:HEAD");
+		parser.addArgument("--movetolastcommit").metavar("MOVETOLASTCOMMIT").required(false) //
```

Comment:
Would be easier if the user just passes `--movetolastcommit` which automatilally means true. Other boolean parameters such as  `--stacktrace` and `--insecure` behave the same way.